src/clj/rems/api/services/catalogue.clj

```diff
@@ -9,16 +9,16 @@
             [rems.db.resource :as resource]
             [rems.db.workflow :as workflow]))
 
-(defn- dependencies-for-catalogue-item [id]
+(defn- dependencies-for-catalogue-item [id user-id]
   (let [{:keys [formid wfid resource-id]} (catalogue/get-localized-catalogue-item id)]
     {:form (form/get-form-template formid)
-     :resource (resource/get-resource resource-id)
+     :resource (resource/get-resource resource-id user-id)
      :workflow (workflow/get-workflow wfid)
      :licenses (licenses/get-licenses {:wfid wfid
                                        :items [id]})}))
 
 (defn create-catalogue-item! [{:keys [localizations organization] :as command} user-id]
-  (or (util/forbidden-organization? user-id organization)
+  (or (util/forbidden-organization-error? user-id organization)
```

Comment:
If this returns an error it should not be with a `?`?

Reply:
Done.